emcc.py

```diff
@@ -2225,19 +2233,16 @@ def dedup_list(lst):
     # exit block 'source transforms'
     log_time('source transforms')
 
-    with ToolchainProfiler.profile_block('memory initializer'):
-      memfile = None
-      if not shared.Settings.MEM_INIT_IN_WASM:
-         memfile = shared.replace_or_append_suffix(target, '.mem')
-      if memfile:
+    if memfile and not shared.Settings.MINIMAL_RUNTIME:
```

Comment:
comment on why not minimal runtime? (not obvious to me. and is it a TODO, or will never be needed?)

Reply:
Added a comment.

From looking at the code it looks like MINIMAL_RUNTIME expected `Module['mem']` to be set and it does this (although only for shell/node environments in `shell_minimal.js` beased on the name of the wasm file.  Therefore so substitution of this kind is needed.